File: seminary/seminary/web_form/enroll_in_a_course/enroll_in_a_course.py
# seminary/seminary/web_form/enroll_in_a_course/enroll_in_a_course.py

import frappe


def get_context(context):
    pass

@frappe.whitelist()
def get_student_name():
    user = frappe.session.user
    student = frappe.get_value("Student", {"user": user}, "student_name")
    print(student)
    return student

File: seminary/seminary/web_form/enroll_in_a_course/enroll_in_a_course.js
frappe.ready(() => {
    // bind events here
    frappe.web_form.after_load = setTimeout(function() {
        frappe.call('get_student_name', self).then(r => {
            const student = r.message;
            console.log("Student: ", student);
            frappe.web_form.set_value('student_ce', student);       
        });
    }, 2000); // 2 seconds delay
});
